### PR TITLE
use status code const to express http status

### PR DESCRIPTION
**Pull request by foxyriver**
**What this PR does / why we need it**:

Using status code const to express http status.

**Special notes for your reviewer**:

**Release note**:

```release-note
```

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37364)

### CODE REVIEW

File: pkg/kubectl/resource/visitor.go [hunk 1]
```diff
@@ -263,12 +263,12 @@ func readHttpWithRetries(get httpget, duration time.Duration, u string, attempts
...
 
 		// Error - Set the error condition from the StatusCode
-		if statusCode != 200 {
+		if statusCode != http.StatusOK {
 			err = fmt.Errorf("unable to read URL %q, server reported %s, status code=%d", u, status, statusCode)
 		}
 
 		if statusCode >= 500 && statusCode < 600 {
-			// Retry 500's
+			// Retry when the StatusCodes is 5**
```
Comment (deads2k) [inline pkg/kubectl/resource/visitor.go]: I prefer the old comment.

**Comment by deads2k** [comment]
nit, then lgtm.  Tag me again when its ready.

**Comment by foxyriver** [comment]
@deads2k its ready

**Comment by deads2k** [comment]
> @deads2k its ready

lgtm, thanks for the cleanup.

**Comment by rmmh** [comment]
@k8s-bot unit test this

**Pull request merged**